src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts

```diff
@@ -35,6 +35,7 @@ export class DotEditContentHtmlService {
     iframe: ElementRef;
     iframeActions$: Subject<any> = new Subject();
     pageModel$: Subject<DotPageContainer[]> = new Subject();
+    mutationConfig = { attributes: false, childList: true, characterData: false };
```

Comment:
This is not the right way to test this... I mean, you are adding a property to a class that doesn't need to exist just to be able to test it.

The right way to test this is to mock the `MutationObserver` and spy that is was called with the right args.

Because I don't know how to do this and I don't think is worth the time right now to do it, please add a TODO in the test so we can revisit this later.